app/models/manageiq/providers/base_manager/operations_worker/runner.rb

```diff
@@ -0,0 +1,17 @@
+class ManageIQ::Providers::BaseManager::OperationsWorker::Runner < ::MiqQueueWorkerBase::Runner
+  OPTIONS_PARSER_SETTINGS = ::MiqWorker::Runner::OPTIONS_PARSER_SETTINGS + [
+    [:ems_id, 'EMS Instance ID', String],
+  ]
+
+  def worker_roles
+    %w[ems_operations"]
```

Comment:
@agrare This is a typo - I think this should actually fail

Reply:
ugh - who merged this! :(

Reply:
:man_facepalming: https://github.com/ManageIQ/manageiq/pull/19680